web/src/contexts/AuthContext.tsx

```diff
@@ -36,5 +83,11 @@ export const AuthProvider = ({ children }: { children: React.ReactNode }) => {
 }
 
 export const useAuth = () => {
-  return useContext(AuthContext)
+  const context = useContext(AuthContext)
+
+  if (context === undefined) {
```

Comment:
why not this?
```suggestion
  if (!context) {
```
did you face a `falsy` issue?